packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "04085c80c2c84ea6b8375042d61797f7f46138fd",
-    "ref_origin" : "dcos-mesos-master-f179400b"
+    "ref": "9d1cb5ecba1a7921b067ad94092c3663adf19b24",
+    "ref_origin" : "dcos-mesos-1.2.0-rc1"
```

Comment:
Thanks for pushing to the mesosphere/mesos dcos-mesos-1.2.0-rc1 branch.
Could you please add the cherry-picks onto that branch as well? Seems like your SHA doesn't match the branch:
```
$ git log --oneline public/dcos-mesos-1.2.0-rc1..9d1cb5ecba1a7921b067ad94092c3663adf19b24
9d1cb5e Updated mesos containerizer to ignore GPU isolator creation failure.
45c0244 Set LIBPROCESS_IP into docker container.
30297e6 Mesos UI: Change paths, pailer, and logs to work with a reverse proxy.
28f8c90 Changed agent_host to expect a relative path.
50b6503 Revert "Fixed the broken metrics information of master in WebUI."
```